### ISSUE TITLE
How do I get a value from openssl config

### ISSUE THREAD

**Issue by beldmit** [openssl/openssl#12467]
I want to get a particular value from the default openssl config from my application.

Is there any API for it now?

The question is both about 1.1.1 and master branches.

**Comment by paulidale** [comment]
Refer: #20940 for a possibility to get values back from providers.  It's not quite the same but close.

**Comment by nhorman** [comment]
it should also be noted that openssl config files are in the ini format, for which there are many libraries to parse independently

Marking as inactive to be closed at the end of 3.4 dev, barring further input

**Comment by beldmit** [comment]
Well... It's not a problem to parse the config. The problem is to find the global state of OpenSSL (originally from the configuration file). And, just to begin, the file itself

**Comment by t8m** [comment]
I do not think this is an actionable RFE. We would need a more concrete specification together with examples as how this would be supposed to work.

**Comment by t8m** [comment]
Closing. This is not actionable. Please open a concrete proposal for an API implementing this functionality.
